docs/security/recovering-from-a-system-compromise.md

```diff
@@ -34,66 +35,65 @@ This is the easiest option, yet the most destructive. It will wipe all of your d
 
-## Using a Second Linode
+## Use a Second Linode
 
 You can use a second Linode for the most seamless transition to a new system.
 
-1.  Begin by adding a new Linode to your account. See the [Getting Started](/docs/getting-started/) guide for instructions.
+1.  Add a new Linode to your account. See the [Getting Started](/docs/getting-started/) guide for instructions.
 2.  Set a strong password for **root** and all user accounts, making sure not to reuse any passwords from the compromised system.
 3.  Upgrade all system packages:
 
     Debian/Ubuntu:
 
-        apt-get update
-        apt-get upgrade --show-upgraded
+        sudo apt update && sudo apt upgrade
 
     CentOS/Fedora:
 
-        yum update
+        sudo yum update
 
-4.  Follow the instructions in the [Securing Your Server](/docs/securing-your-server/) guide to disable root logins via SSH, or to disable password logins, in favor of SSH keys only, for all accounts.
+4.  Follow the instructions in the [Securing Your Server](/docs/securing-your-server/) guide to harden SSH access and activate the firewall.
 
 ### Rebuild Your Configuration
 
 Rebuild your production server's configuration on the new Linode.
 
 -   Install updated versions of all applications
--   Restrict access to all applications appropriately, using firewall software or any other methods deemed appropriate
+-   Restrict access to all applications
 -   Assign new administrative passwords for all web applications
 
 ### Copy and Audit Your Data
 
-The next task is to copy your data to the new Linode, and make sure you've purged all compromised portions. Follow these steps:
+The next task is to copy your data to the new Linode, and make sure that all compromised portions have been purged.
 
 1.  Create a temporary directory on the new Linode.
-2.  Copy any needed user and configuration data from the compromised Linode using [rsync](/docs/tools-reference/tools/introduction-to-rsync/) or `scp`. If you are not familiar with these programs, you can find more information by entering the `man rsync` or `man scp` commands.
+2.  Copy any needed user and configuration data from the compromised Linode using [rsync](/docs/tools-reference/tools/introduction-to-rsync/) or `scp`.
 
     {{< caution >}}
 Do not log in to the new Linode from the compromised Linode. Files should be pulled from the compromised server to your new setup instead.
 {{< /caution >}}
 
-3.  Audit your data using tools such as `rkhunter` and `clamav`. You may wish to use additional malware scanners as well to be certain you aren't retaining tainted files. Examine all system scripts manually for contaminated code, and replace all suspicious executable files with known good copies.
+3.  Audit your data using tools such as `rkhunter` and `clamav`. You can use additional malware scanners to be certain you aren't retaining tainted files. Examine all system scripts manually for contaminated code, and replace all suspicious executable files with known good copies.
```

Comment:
There is a ClamAV guide now: https://www.linode.com/docs/security/vulnerabilities/scanning-your-linode-for-malware/